build.gradle

```diff
@@ -180,7 +180,7 @@ jacocoTestCoverageVerification {
                 counter = "INSTRUCTION"
                 value = "COVEREDRATIO"
                 // TODO: in the future the following value should become higher than 0.92
-                minimum = 0.895
```

Comment:
was this intended ?

Reply:
The change is pushed by @bassner and it was intended.